ghost/signup-form/package.json

```
@@ -73,6 +73,7 @@
         "tailwindcss": "3.3.2",
         "typescript": "5.0.4",
         "vite": "4.3.9",
+        "vite-plugin-commonjs": "^0.7.1",
```

```suggestion
        "vite-plugin-commonjs": "0.7.1",
```

+ re-run `yarn` please 😀